src/XamlStyler/Options/StylerOptions.cs

```diff
@@ -298,6 +298,14 @@ public StylerOptions(string config = "")
         [JsonIgnore]
         public bool SearchToDriveRoot { get; set; }
 
+        [Category("Attribute Reordering")]
+        [DisplayName("Ignore Design-Time Reference Prefix")]
+        [JsonProperty("IgnoreDesignTimeReferencePrefix", DefaultValueHandling = DefaultValueHandling.Populate)]
```

Comment:
Why not ```DefaultValueHandling.IgnoreAndPopulate```?

Reply:
Wow, I missed that. It's better of course :)